Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Silicon.yaml

```diff
@@ -1753,11 +1760,27 @@
                      Enable- Skip Full CD clock initializaton, Disable(Default)- Initialize the full CD clock if not initialized by Gfx PEIM
       length       : 0x01
       value        : 0x0
+  - EnableTimedGpio0 :
+      name         : Timed GPIO 0
+      type         : Combo
+      option       : $EN_DIS
+      help         : >
+                     When FALSE, it disables Timed GPIO 0.
+      length       : 0x01
+      value        : 0x0
+  - EnableTimedGpio1 :

```

Comment:
one bit should be good enough for enable/disable.
length : 1b   ->  This will use 1 bit instead of 1 byte.

Reply:
Done changing it, thanks for the review.